apps/challenges/views.py

```diff
@@ -2553,3 +2555,77 @@ def validate_challenge_config(request, challenge_host_team_pk):
         message = "Challenge config validation success"
         response_data = {"Success": message}
         return Response(response_data, status=status.HTTP_200_OK)
+
+
+@api_view(["GET"])
+@throttle_classes([UserRateThrottle])
+@permission_classes((permissions.IsAuthenticated, HasVerifiedEmail))
+@authentication_classes((ExpiringTokenAuthentication,))
+def get_worker_logs(request, challenge_pk):
+    if not is_user_a_host_of_challenge(request.user, challenge_pk):
+        response_data = {
+            "error": "Sorry, you are not authorized to access the worker logs."
+        }
+        return Response(response_data, status=status.HTTP_400_BAD_REQUEST)
+
+    challenge = challenge = Challenge.objects.get(pk=challenge_pk)
+    response_data = []
+
+    log_group_name = "challenge-pk-{}-workers".format(challenge_pk)
+    log_stream_prefix = challenge.queue
+    pattern = "WORKER_LOG"
+
+    # This is to specify the time window for fetching logs. Currently it's 15 minutes before from current time.
+    timeframe = 15
+    current_time = int(round(time.time() * 1000))
+    start_time = current_time - timeframe * 900000
+    end_time = current_time
+
+    logs = get_logs_from_cloudwatch(log_group_name, log_stream_prefix, start_time, end_time, pattern)
+
+    response_data = {"logs": logs}
+    return Response(response_data, status=status.HTTP_200_OK)
+
+
+@api_view(["PUT"])
+@throttle_classes([UserRateThrottle])
+@permission_classes((permissions.IsAuthenticated, HasVerifiedEmail))
+@authentication_classes((ExpiringTokenAuthentication,))
+def manage_worker(request, challenge_pk, action):
+    if not is_user_a_host_of_challenge(request.user, challenge_pk):
+        response_data = {
+            "error": "Sorry, you are not authorized."
+        }
+        return Response(response_data, status=status.HTTP_400_BAD_REQUEST)
+
+    challenge = Challenge.objects.get(pk=challenge_pk)
+
+    response_data = {}
+
+    if action == 'start':
+        response = start_workers([challenge])
+        count, failures = response["count"], response["failures"]
+        logging.info("Count is {} and failures are: {}".format(count, failures))
+        if count:
+            response_data = {"action": "Success"}
```

Comment:
`response_data = {"action": "Worker(s) is successfully started"}`

Reply:
Handling the message dynamically in the frontend. This is just to denote the success of whatever action was chosen.

Reply:
okay!